stdlib/heapq.pyi

```diff
@@ -1,14 +1,12 @@
+from _heapq import *
 from _typeshed import SupportsRichComparison
 from typing import Any, Callable, Iterable, TypeVar
 
-_T = TypeVar("_T")
+_S = TypeVar("_S")
 
-def heappush(__heap: list[_T], __item: _T) -> None: ...
-def heappop(__heap: list[_T]) -> _T: ...
-def heappushpop(__heap: list[_T], __item: _T) -> _T: ...
-def heapify(__heap: list[Any]) -> None: ...
-def heapreplace(__heap: list[_T], __item: _T) -> _T: ...
-def merge(*iterables: Iterable[_T], key: Callable[[_T], Any] | None = ..., reverse: bool = ...) -> Iterable[_T]: ...
-def nlargest(n: int, iterable: Iterable[_T], key: Callable[[_T], SupportsRichComparison] | None = ...) -> list[_T]: ...
-def nsmallest(n: int, iterable: Iterable[_T], key: Callable[[_T], SupportsRichComparison] | None = ...) -> list[_T]: ...
+__about__: str
```

Comment:
Is this needed? Usually `__foo__` names are treated as public, not private like `_foo` names.

Reply:
I think it is, or at least when I checked earlier it appears dunders are not imported using `from x import *`

Reply:
Lol, cpython developers have decided to copy/paste the same text into both modules

```
>>> from _heapq import __about__ as a
>>> from heapq import __about__ as b
>>> a == b
True
>>> a is b
False
```